### PR TITLE
Merge/release 7.62.1 into 7.63

### PR DESCRIPTION
**Pull request by mebarbosa**

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This PR has the <code>releases</code> label: some checks will be skipped.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by MiSikora — CHANGES_REQUESTED** [review]
Some files that shouldn't be included were added due to no conflicts but they were removed in `7.63`.

File: modules/features/widgets/src/main/res/layout/non_scaling_text.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
```
[the remark this answers is not in the document]
Comment (mebarbosa) [reply modules/features/widgets/src/main/res/layout/non_scaling_text.xml]: Deleted here: [5ccf7bc](https://github.com/Automattic/pocket-casts-android/pull/2141/commits/5ccf7bc69d8941203c650a72dfafff93e92c3a4e)

File: modules/features/widgets/src/main/res/layout/non_scaling_text_bold.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
```
[the remark this answers is not in the document]
Comment (mebarbosa) [reply modules/features/widgets/src/main/res/layout/non_scaling_text_bold.xml]: Deleted here: [5ccf7bc](https://github.com/Automattic/pocket-casts-android/pull/2141/commits/5ccf7bc69d8941203c650a72dfafff93e92c3a4e)

File: modules/features/widgets/src/main/res/layout/non_scaling_text_dynamic.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
```
[the remark this answers is not in the document]
Comment (mebarbosa) [reply modules/features/widgets/src/main/res/layout/non_scaling_text_dynamic.xml]: Deleted here: [5ccf7bc](https://github.com/Automattic/pocket-casts-android/pull/2141/commits/5ccf7bc69d8941203c650a72dfafff93e92c3a4e)

File: modules/features/widgets/src/main/res/layout/non_scaling_text_dynamic_bold.xml [hunk 4]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```
[the remark this answers is not in the document]
Comment (mebarbosa) [reply modules/features/widgets/src/main/res/layout/non_scaling_text_dynamic_bold.xml]: Deleted here: [5ccf7bc](https://github.com/Automattic/pocket-casts-android/pull/2141/commits/5ccf7bc69d8941203c650a72dfafff93e92c3a4e)

**Review by MiSikora — APPROVED** [review]

**Pull request merged**